go.mod

```diff
@@ -54,11 +54,11 @@ require (
 	google.golang.org/protobuf v1.35.1
 	gopkg.in/yaml.v2 v2.4.0
```

Comment:
Do we need those changes? I think they can be dropped

Reply:
I guess these are done due to local setup, it is not required for working. I will drop them.